app/templates/main/accessibility.html

```diff
@@ -0,0 +1,260 @@
+{% extends "base.html" %}
```

Comment:
Can we remove any pages like this we are not currently linking to?

Reply:
Yes, we can, but the govukFooter macro relies on there being a path for accessibility, cookies, and privacy. If they are removed, it will no longer work. I can remove all of the information for those three pages and just leave them blank, but still have them as roots, or alter the actual macro itself. Which would be preferable?